### PR TITLE
i18n(zh-cn): update `guides/integrations-guide/vercel.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

update `guides/integrations-guide/vercel.mdx` since #4690, #4713

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -176,6 +203,32 @@ import astroLogo from '../assets/logo.png';
...
 ```
 
+### devImageService
+
+**类型：** `'sharp' | 'squoosh' | string`<br/>
+**适用于：** Serverless, Static
+**添加于：** `@astrojs/vercel@3.3.0`
+**默认值：** 'sharp'
+
+Allows you to configure which image service to use in development when [imageService](#imageservice) is enabled. This can be useful if you cannot install Sharp's dependencies on your development machine, but using another image service like Squoosh would allow you to preview images in your dev environment. Build is unaffected and will always use Vercel's Image Optimization.
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx:213]:
```suggestion
```

Remove source text.

**Review by Genteure — CHANGES_REQUESTED** [review]

**Comment by 100gle** [comment]
There are some updates from upstream #4713, please review again. @Genteure

File: src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx [hunk 2]
```diff
@@ -141,7 +168,7 @@ export default defineConfig({
 **适用于：** Serverless, Static<br/>
 **添加于：** `@astrojs/vercel@3.3.0`
 
-启用后，在生产中会自动配置并使用由 Vercel Image Optimization API 提供支持的[图像服务](/zh-cn/reference/image-service-reference/) 。在开发中，将使用内置的 Squoosh-based 服务。
+启用后，在生产中会自动配置并使用由 Vercel Image Optimization API 提供支持的[图像服务](/zh-cn/reference/image-service-reference/) 。在开发中，将使用由[`devImageService`](#devimageservice) 指定的图像服务。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx:171]:
```suggestion
启用后，在生产中会自动配置并使用由 Vercel Image Optimization API 提供支持的[图像服务](/zh-cn/reference/image-service-reference/) 。在开发中，将使用由 [`devImageService`](#devimageservice) 指定的图像服务。
```

nit: formatting

**Review by Genteure — APPROVED** [review]
LGTM 👍

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**